jf_agent/jf_jira/__init__.py

```diff
@@ -302,11 +303,18 @@ def download_and_write_issues_needing_re_download():
             config.compress_output_files,
             download_statuses(jira_connection),
         )
+        try:
+            write_file(
+                config.outdir,
+                'jira_teams',
+                config.compress_output_files,
+                download_teams(jira_connection)
+            )
+        except Exception as e:
+            logging_helper.log_standard_error(logging.DEBUG, msg_args=[e], error_code=0000, exc_info=True)
```

Comment:
Nit: Change to logger.debug

Reply:
https://github.com/Jellyfish-AI/jf_agent/pull/284/commits/1d1056a4b988b271aa79d69242f543089d93512a